plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx

```diff
@@ -323,9 +326,9 @@ export const ActionsPage = () => {
   return (
     <Page themeId="home">
       <Header
-        pageTitleOverride="Create a New Component"
-        title="Installed actions"
-        subtitle="This is the collection of all installed actions"
+        pageTitleOverride={t('create_a_new_component')}
+        title={t('installed_actions')}
+        subtitle={t('this_is_the_collection_of_all_installed_actions')}
```

Comment:
I wonder if we should be making these more specific to their placement rather than this.

`installed_actions_subtitle` or something like that? 

`_title` and `_subtitle` seem to be nice patterns I think